specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/InfrastructureInsights.json

```diff
@@ -0,0 +1,336 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2016-05-01",
+        "title": "InfrastructureInsightsManagementClient",
+        "description": "The Admin Infrastructure Insights Management Client."
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/providers/Microsoft.InfrastructureInsights.Admin/operations": {
+            "get": {
+                "x-ms-examples": {
+                    "Returns a list of support REST operations.": {
+                        "$ref": "./examples/Operations/List.json"
+                    }
+                },
+                "description": "Returns a list of support REST operations.",
+                "tags": [
+                    "InfrastructureInsights"
+                ],
+                "operationId": "Operations_List",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                },
+                "produces": [
+                    "application/json"
+                ],
+                "consumes": [
+                    "application/json"
+                ]
+            }
+        }
+    },
+    "definitions": {
```

Comment:
>definitions [](start = 5, length = 11)

There are a bunch of definitions that are not being used on this file, are they being used at all?